specification/quota/Quota.Management/GroupQuotaLimitList.tsp

```diff
@@ -0,0 +1,127 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+import "./GroupQuotasEntity.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace Microsoft.Quota;
+/**
+ * List of Group Quota Limit details.
+ */
+@parentResource(GroupQuotasEntity)
+model GroupQuotaLimitList
+  is Azure.ResourceManager.ProxyResource<GroupQuotaLimitListProperties> {
+  ...ResourceNameParameter<
+    Resource = GroupQuotaLimitList,
+    KeyName = "location",
+    SegmentName = "groupQuotaLimits",
+    NamePattern = ""
+  >;
+}
+
+/**
+ * Virtual resource representing a resource provider scope within a group quota.
+ */
+@parentResource(GroupQuotasEntity)
+@armVirtualResource
+model Virtual {
+  /**
+   * The resource provider name, such as - Microsoft.Compute. Currently only Microsoft.Compute resource provider supports this API.
+   */
+  #suppress "@azure-tools/typespec-azure-core/key-visibility-required" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+  @key
+  @path
+  @segment("resourceProviders")
+  @pattern("^((?!-)[a-zA-Z0-9-]{1,63}(?<!-)\\.)+[a-zA-Z]{2,63}$")
+  resourceProviderName: string;
+}
+
+alias GroupQuotaLimitListsOps = Azure.ResourceManager.Legacy.ExtensionOperations<
```

Comment:
Why do we need this alias? If you want to rename, `Extension.ManagementGroup` can accept parameter name as parameter.